tonic/models/vic/grid_params.py

```diff
@@ -661,64 +1218,172 @@ def grid_params(soil_dict, target_grid, snow_dict, veg_dict, veglib_dict,
                 for y, x in pyzip(ymask, xmask):
                     out_dict[var][:, :, y, x] = fill_val
 
+            elif mydict[var].ndim == 4:
+                j = mydict[var].shape[1]
+                k = mydict[var].shape[2]
+                l = mydict[var].shape[3]
+                out_dict[var] = np.ma.zeros((j, k, l, ysize, xsize),
+                                            dtype=dtype)
```

Comment:
No I totally agree. I think it's way better for version control to separate out PRs. Since this is a new feature it should be a standalone PR without any other overhauls.